configurations/configurations_test.go

```diff
@@ -0,0 +1,32 @@
+package configurations
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var expectedGenericOutput = `FROM golang:latest As golang
```

Comment:
why "As golang"?

Reply:
`As` is not needed here. I was just testing whether all the fields of template are working or no.